### ISSUE TITLE
How to catch "- (void)session:(QBRTCSession *)session disconnectedByTimeoutFromUser:(NSNumber *)userID" method

### ISSUE THREAD

**Issue by Shamsiddin** [QuickBlox/quickblox-ios-sdk#678]
**Environment details**
iOS version: 9.3.2
Quickblox iOS SDK version: 2.7.6
QuickbloxWebRTC SDK version: 2.2
Language: Objective-C
XCode version: 8.1

**Any others comments?**
I know it's not bug, but I have no clue how to test/reproduce/catch the `- (void)session:(QBRTCSession *)session disconnectedByTimeoutFromUser:(NSNumber *)userID` method. [Official documentation](http://quickblox.com/developers/Sample-webrtc-ios#Set_disconnect_time_interval) says:

> After a disconnect from an opponent happened we are starting timer and waiting for a given time interval in case connection establishing/reconnecting again.
> Default value: 30 seconds 

I even set `disconnectTimeInterval` to `3` to make this `- (void)session:(QBRTCSession *)session disconnectedByTimeoutFromUser:(NSNumber *)userID` method catch, but still haven't catch it. 
Any ideas how can I make it happen?

**Comment by Raikerian** [comment]
Hi @Shamsiddin 

Value of disconnectTimeInterval can't be lower than 10. We will add this in documentation for the next release 2.3 expected near the end of the year.

**Comment by Shamsiddin** [comment]
Hi @Raikerian

I'll test it with 10 seconds `disconnectTimeInterval`. Can You please give me any scenario to catch this method, because before I tested it with 15 seconds, and wasn't any success.

Actually my scenario is:
1. user **A** called to user **B**
2. user **B** accepted the call and both sides started to talk each other.
3. Then user **B** went offline or killed the application
4. user **A** received signals in this order
```
- (void)session:(QBRTCSession *)session disconnectedFromUser:(NSNumber *)userID
- (void)session:(QBRTCSession *)session connectionFailedForUser:(NSNumber *)userID
- (void)session:(QBRTCSession *)session connectionClosedForUser:(NSNumber *)userID
- (void)sessionDidClose:(QBRTCSession *)session
```


but didn't catch the `- (void)session:(QBRTCSession *)session disconnectedByTimeoutFromUser:(NSNumber *)userID` method. I don't know what to do to make it happen.

Thanks.

**Comment by Shamsiddin** [comment]
Hi @Raikerian 
Please check my last comment with detailed info. What's wrong with `disconnectTimeInterval = 15`
interval? I'm still having this issue.

**Comment by Raikerian** [comment]
@Shamsiddin 

I have looked more deep into this with a team, and it was decided that this property will be gone in next release. Reasons: old versions of webrtc were having very long time for disconnection before ice connection was failed. Now it is all managed automatically by webrtc, and everything will be performed much faster.
As for reasons why it is not working in 2.2, I don't have any real explanation cause that version has lots of bugs, and every one of them should be gone in 2.3 (expected near the end of the year).

**Comment by Raikerian** [comment]
We have just released our new WebRTC SDK 2.3 which deprecating this method.
Here is full changelog of version 2.3: http://quickblox.com/developers/Sample-webrtc-ios#Framework_changelog

Anyway, if you will have any other problems or questions, feel free to start new issue to let us know.

Have a nice holidays.
